e2e_tests/tests/requirements.txt

```diff
@@ -1,4 +1,5 @@
 appdirs
+protobuf<3.20  # TODO(MLG-336): remove this requirement when tensorflow-macos is upgraded.
```

Comment:
since it's tied to macos, and tensorflow==2.8.4 users won't have the same problem, maybe be a bit more liberal and only pin it for macos?

```suggestion
# TODO(MLG-336): remove this requirement when tensorflow-macos is upgraded.
protobuf<3.20; sys_platform == 'darwin' and platform_machine == 'arm64'
```

Reply:
Sure